### PR TITLE
Complete user story 11

### PR DESCRIPTION
**Pull request by kbergstrom78**

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class AdminController < ApplicationController
   def shelters_index
     @shelters = Shelter.order_by_name_desc
+    @pending_apps = Shelter.joins(pets: [{ application_pets: :application }]).where('applications.status' == "Pending")
```
Comment (Jredish11) [inline app/controllers/admin_controller.rb:4]: going to have to walk through this one with me hah.  Im going to take a shot at it though.  we want use applications_pets table to join our pets and application tables.  then its going to select applications with status pending and put them in our Shelter table which allows it to render on the shelters index?

Comment (kbergstrom78) [reply app/controllers/admin_controller.rb:4]: Yes!  So, building off of what Juliet said during our checkin...
we're taking the shelter table and joining it with the pets table to get data for all of the pets in a shelter.  But because of the many-many relationship, we have to go through the application_pets - that's where that nested join comes in that Juliet was talking about.  

This part was the tricky part for me: __pets: [{ application_pets: :application }]:_  but what it's doing is retrieving the shelters that have pets with pending applications.  So it's joining Pets to the applications by going through the application_pets table.  At least I think (hope) that that's what it is doing.  

TL:DR - it's joining three tables and then filtering by status so that we're only seeing shelters that have pending applications.
